R/constants.R

```diff
@@ -31,6 +32,14 @@ GTAP_HISTORICAL_YEAR <- 2000
 aglu.GLU <- "GLU"
 aglu.GLU_NAME_DELIMITER <- ""  # delimiter between the GLU name and number
 
+# cotton seed and lint conversion
+CONV_COTTON_LINT <- 0.4
+
+# Price conversion from alfalfa to grass hay
+PRICERATIO_GRASS_ALFALFA <- 0.7
```

Comment:
And the conversion from "seed cotton" to "cotton lint" is just standard FAO stuff. Their total production volume of "seed cotton" (the crop we map to FiberCrop) is about 40% cotton lint and 60% cotton seeds. According to the USDA cost sheets, the lint accounts for over 90% of the production revenue. Anyway the FAO PRICESTAT database disaggregates the seed and the lint as different commodities, and doesn't report a producer price for "seed cotton". The steps done here are trying to get a producer price for seed cotton from the reported cotton lint and cotton seed prices. The 60/40 comes from the metadata to the FAOSTAT classic version, so any URL in any documentation is no longer valid. Will try to re-post documentation shortly.